### PR TITLE
Support Spark bin function

### LINKED ISSUE #4930: Support Spark bin function
### Description

Spark support [bin](https://spark.apache.org/docs/latest/api/sql/#bin) since 1.5.0, let's add it to Velox.

### PR DESCRIPTION
**Pull request by Yohahaha**
Support Spark bin function, this function will return the string representation of the long input value represented in binary.

Fixed #4930

### CODE REVIEW

**Comment by Yohahaha** [comment]
Hi @mbasmanova could you help take a look?

**Review by mbasmanova — COMMENTED** [review]
@bikramSingh91 Bikram, would you help review this PR?

File: velox/functions/sparksql/tests/ArithmeticTest.cpp [hunk 1]
```diff
@@ -224,5 +224,20 @@ TEST_F(CeilFloorTest, Limits) {
...
+ protected:
+  std::optional<std::string> bin(std::optional<std::int64_t> arg) {
+    return evaluateOnce<std::string, int64_t>("bin(c0)", {arg}, {BIGINT()});
+  }
+};
+
+TEST_F(BinTest, bin) {
+  EXPECT_EQ(bin(std::nullopt), std::nullopt);
+  EXPECT_EQ(bin(13), "1101");
+  EXPECT_EQ(
```
Comment (bikramSingh91) [inline velox/functions/sparksql/tests/ArithmeticTest.cpp:237]: can you add two more cases, one for 0(all 0s) and another for int64_max (all 1s)

**Review by bikramSingh91 — CHANGES_REQUESTED** [review]

File: velox/functions/sparksql/tests/ArithmeticTest.cpp (hunk 1 above)
Comment (Yohahaha) [reply velox/functions/sparksql/tests/ArithmeticTest.cpp:237]: Already add these cases, thanks!

**Review by bikramSingh91 — APPROVED** [review]
thank you

**Comment by Yohahaha** [comment]
Hi @bikramSingh91 do I need fix conflicts?

**Comment by Yohahaha** [comment]
Hi @bikramSingh91, could we merge this PR?

**Comment by Yohahaha** [comment]
Hi @bikramSingh91 @Yuhta , could we merge this PR? is problem I need to solve?  thanks!

**Comment by bikramSingh91** [comment]
@Yohahaha Thanks for following up. Will get this in today.

**Pull request closed without merging**